static/src/stylesheets/module/facia/snaps/_football.scss

```diff
@@ -231,10 +232,6 @@ $footballBadgeSizeDesktop: 120px;
 
 }
 
-.facia-snap.facia-snap--football.facia-snap-embed {
-    height: auto !important; // height is normally set on the parent snap element in Facia
```

Comment:
ugh. I hate `!important`

Reply:
Yeah you can blame past Nic for that one 😞

Reply:
It is used [a lot](https://github.com/guardian/frontend/search?l=SCSS&q=%21important) within frontend at the moment unfortunately!